tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -6767,7 +6767,7 @@ TEST_F(EnergyPlusFixture, VSCoilUnitary_NoNegativeCapacity)
     Real64 constexpr SpeedRatio{0.0};
     int constexpr SpeedCal{1};
     VariableSpeedCoils::InitVarSpeedCoil(*state, DXCoilNum, SensLoad, LatentLoad, fanOp, OnOffAirFlowRatio, SpeedRatio, SpeedCal);
-    EXPECT_TRUE(compare_err_stream_substring("entering air temperature is less than design outlet air temperature", true));
+    EXPECT_TRUE(compare_err_stream_substring("This will yield negative coil capacity sizing", true));
```

Comment:
This unit test should fail. The old message used "will" and the new warning message above uses:

    This would yield negative coil capacity sizing